### PR TITLE
Button titles (tooltips)

### PR DESCRIPTION
**Pull request by grant-davidson**
iFbQaqrSO6

### CODE REVIEW

File: src/exportToHtml.ts [hunk 1]
```diff
@@ -75,12 +75,12 @@ export const exportToHtml = async (scenes: Scene[]) => {
...
             <div id="navigation">
-              <button class="navbutton" type="button" onClick="moveLeft()">&#9664;</button>
+              <button class="navbutton" type="button" onClick="moveLeft()" title="Previous slide">&#9664;</button>
               <div id="title">1 of ${scenes.length}</div>
-              <button class="navbutton" type="button" onClick="moveRight()">&#9654;</button>          
+              <button class="navbutton" type="button" onClick="moveRight()" title="Next slide">&#9654;</button>          
               <div id="rightbuttons">
-                <button type="button" onClick="toggleMaximise()">&#x26F6</button>
-                <button type="button" onClick="closeNavigation()">&#x2716;</button>
+                <button type="button" onClick="toggleMaximise()" title="Toggle full-screen">&#x26F6</button>
```
Comment (dai-shi) [inline src/exportToHtml.ts:82]:
```suggestion
                <button type="button" onClick="toggleMaximise()" title="Toggle full-screen">&#x26F6;</button>
```

Actually, my browser (Chrome) didn't show the correct icon before. Does this fix it?

Comment (grant-davidson) [reply src/exportToHtml.ts]: Not sure, it works without the ; in my Chrome browser.  Can you check the new version?

Comment (grant-davidson) [reply src/exportToHtml.ts]: sorry, forgot to use the "Commit suggestion" button

**Comment by dai-shi** [comment]
image

Hmm, it doesn't. Maybe font? Never mind for now.

**Review by dai-shi — APPROVED** [review]
Thanks for the improvement!

**Comment by grant-davidson** [comment]
It seems that symbol is font dependent.  I'll see if I can find something similar the available in all fonts

**Pull request merged**